doc/source/tune-searchalg.rst

```diff
@@ -103,6 +103,31 @@ An example of this can be found in `sigopt_example.py <https://github.com/ray-pr
     :show-inheritance:
```

Comment:
BTW, can you also update `doc/source/tune.rst` to better showcase this on the front-page of the project?